spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -439,32 +439,63 @@ class PerfRunHelperClass
     end
   end
   describe "#copy_system_logs" do
-    let!(:hosts) do
-      [Beaker::Host.create("master",
-                           { platform: Beaker::Platform.new("centos-6.5-x86_64"),
-                             role: "master" }, logger: @logger)]
+    let(:host) { double.as_null_object }
+    let(:archive_root) { "/tmp/test/archive" }
+    let(:archive_dir) { "/var/log" }
+    let(:archive_name) { "puppet_logdir.tgz" }
+    let(:archive_path) { File.join(archive_dir, archive_name) }
+
+    context "when archiving succeeds" do
```

Comment:
Running this actually creates the `/tmp/test/archive/#[Double (anonymous)]` on my laptop.

Reply:
Thanks @billclaytor I have added a test for the mkdir_p expectation.